polynote-kernel/src/main/scala/polynote/kernel/context/GlobalInfo.scala

```diff
@@ -1,15 +1,82 @@
-package polynote.kernel.util
+package polynote.kernel.context
 
 import java.io.File
 import java.net.URL
 
-import scala.reflect.internal.util.{AbstractFileClassLoader, BatchSourceFile}
+import cats.effect.IO
+import polynote.kernel.lang.LanguageKernel
+import polynote.kernel.util._
+
 import scala.reflect.internal.util.ScalaClassLoader.URLClassLoader
+import scala.reflect.internal.util.{AbstractFileClassLoader, BatchSourceFile}
 import scala.reflect.io.{AbstractFile, VirtualDirectory}
 import scala.tools.nsc.Settings
 import scala.tools.nsc.interactive.Global
 
-final case class GlobalInfo(global: Global, classPath: List[File], classLoader: AbstractFileClassLoader)
+final case class GlobalInfo(global: Global, classPath: List[File], classLoader: AbstractFileClassLoader) {
```

Comment:
`GlobalInfo` is now everywhere, and it got all the important bits from RuntimeSymbolTable